### PR TITLE
[pr] Removing conflicting crates

### PR DESCRIPTION
**Pull request by DudessaPr**
#### Problem

Cannot compile riscv in svm rollup because of zstd, lz4 and bzip2
Also had issues with solana-perf, accountsdb and symlink in solana-runtime

#### Summary of Changes

Removed zstd, lz4 and bzip2 from runtime, account decoder and accounts db
Removed solana-perf
Fixed accountsdb for solana-runtime
Removed symlink from solana-runtime

Fixes #

### CODE REVIEW

**Review by neutrinoks — APPROVED** [review]
👍

File: accounts-db/src/tiered_storage/hot.rs [hunk 1]
```diff
@@ -69,7 +69,8 @@ pub(crate) const HOT_ACCOUNT_ALIGNMENT: usize = 8;
 pub(crate) const HOT_BLOCK_ALIGNMENT: usize = 8;
 
 /// The maximum supported offset for hot accounts storage.
-const MAX_HOT_ACCOUNT_OFFSET: usize = u32::MAX as usize * HOT_ACCOUNT_ALIGNMENT;
+// const MAX_HOT_ACCOUNT_OFFSET: usize = u32::MAX as usize * HOT_ACCOUNT_ALIGNMENT;
+const MAX_HOT_ACCOUNT_OFFSET: usize = (u32::MAX as u64 * HOT_ACCOUNT_ALIGNMENT as u64) as usize;
```
Comment (Yiwen-Gao) [inline accounts-db/src/tiered_storage/hot.rs:73]: I'm curious why you need to change this here?

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -114,7 +114,6 @@ frozen-abi = [
     "solana-accounts-db/frozen-abi",
     "solana-compute-budget/frozen-abi",
     "solana-cost-model/frozen-abi",
-    "solana-perf/frozen-abi",
```
Comment (Yiwen-Gao) [inline runtime/Cargo.toml:117]: Do you want to comment this instead of deleting it like the other removals?

Comment (DudessaPr) [reply runtime/Cargo.toml:117]: Sure, thanks for mentioning it. I forgot

File: accounts-db/src/tiered_storage/hot.rs (hunk 1 above)
Comment (DudessaPr) [reply accounts-db/src/tiered_storage/hot.rs:73]: I got an error about it, something related to overflow of size

```sov-prover-guest-celestia:    Compiling spl-tlv-account-resolution v0.6.3
sov-prover-guest-celestia:    Compiling spl-token-metadata-interface v0.3.3
sov-prover-guest-celestia:    Compiling spl-transfer-hook-interface v0.6.3
sov-prover-guest-celestia:    Compiling spl-token-2022 v3.0.2
sov-prover-guest-celestia: error[E0080]: evaluation of constant value failed
sov-prover-guest-celestia:   --> /Users/dulatrakhymkul/.cargo/git/checkouts/agave-63bb32cf49464624/0b7ae2f/accounts-db/src/tiered_storage/hot.rs:72:39
sov-prover-guest-celestia:    |
sov-prover-guest-celestia: 72 | const MAX_HOT_ACCOUNT_OFFSET: usize = u32::MAX as usize * HOT_ACCOUNT_ALIGNMENT;
sov-prover-guest-celestia:    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempt to compute `usize::MAX * 8_usize`, which would overflow
sov-prover-guest-celestia: 
sov-prover-guest-celestia: note: erroneous constant encountered
sov-prover-guest-celestia:    --> /Users/dulatrakhymkul/.cargo/git/checkouts/agave-63bb32cf49464624/0b7ae2f/accounts-db/src/tiered_storage/hot.rs:119:21
sov-prover-guest-celestia:     |
sov-prover-guest-celestia: 119 |         if offset > MAX_HOT_ACCOUNT_OFFSET {
sov-prover-guest-celestia:     |                     ^^^^^^^^^^^^^^^^^^^^^^
sov-prover-guest-celestia: 
sov-prover-guest-celestia: note: erroneous constant encountered
sov-prover-guest-celestia:    --> /Users/dulatrakhymkul/.cargo/git/checkouts/agave-63bb32cf49464624/0b7ae2f/accounts-db/src/tiered_storage/hot.rs:122:17
sov-prover-guest-celestia:     |
sov-prover-guest-celestia: 122 |                 MAX_HOT_ACCOUNT_OFFSET,
sov-prover-guest-celestia:     |                 ^^^^^^^^^^^^^^^^^^^^^^
sov-prover-guest-celestia: 
sov-prover-guest-celestia: For more information about this error, try `rustc --explain E0080`.
sov-prover-guest-celestia: error: could not compile `solana-accounts-db` (lib) due to 1 previous error
sov-prover-guest-celestia: warning: build failed, waiting for other jobs to finish...
error: failed to run custom build command for `risc0 v0.3.0 (/Users/dulatrakhymkul/Documents/GitHub/svm-rollup/crates/rollup/provers/risc0)`
```

**Review by neutrinoks — APPROVED** [review]
Left one comment about the conversion issue 👍

File: accounts-db/src/tiered_storage/hot.rs (hunk 1 above)
Comment (Yiwen-Gao) [reply accounts-db/src/tiered_storage/hot.rs:73]: @DudessaPr did you end up trying Michael's suggestion here?

Comment (DudessaPr) [reply accounts-db/src/tiered_storage/hot.rs:73]: Yes, it did not work
For some reason compiler complains
As long as it works correctly I think it is better to leave it like that

**Pull request merged**